Feedback

InesHerrmann opened the pull request:
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `master` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `master` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `master`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://classroom.github.com/help/leaving-feedback-in-github)”.
</details>


Subscribed: @NoranWinterwolf

InesHerrmann committed:
Setting up GitHub Classroom Feedback

NoranWinterwolf committed:
styling

NoranWinterwolf committed:
javascript and final style

NoranWinterwolf committed:
playable with keyboard

NoranWinterwolf committed:
styling

NoranWinterwolf committed:
minor style

NoranWinterwolf committed:
responsive flex

NoranWinterwolf committed:
styling

NoranWinterwolf committed:
add underline for user input

NoranWinterwolf committed:
fix new game button

NoranWinterwolf committed:
debug newGame

NoranWinterwolf committed:
fix

RebeccaRotti commented on game/CSS/style.css:
```diff
@@ -0,0 +1,189 @@
+    padding: 1rem;
+    background-color: grey;
+    border: 10px outset gray;
+    font-size: 520%;
```
solltest im body eine font-size deklarieren und hier dann eventuell statt mit einem so hohen wert zu arbeiten mit vw oder vh arbeiten...

RebeccaRotti commented on game/CSS/style.css:
```diff
@@ -0,0 +1,189 @@
+
+section.game button {
+    color: rgb(61, 69, 78);
+    font-size: 300%;
```
siehe vorheriger kommentar

RebeccaRotti commented on game/JS/script.js:
```diff
@@ -0,0 +1,243 @@
+    outputString += "<span ";
+    for (i = 0; i < code.length; i++) {
+        if (code[i] == userInput[i]) {
+            outputString += "style='color:rgb(96, 192, 0); text-shadow: 0 0 10px greenyellow, 0 0 30px green;'>" + userInput[i];
```
wie schon erwähnt hier eventuell mit klassen statt mit inline styling arbeiten

RebeccaRotti commented on game/JS/script.js:
```diff
@@ -0,0 +1,243 @@
+    buttons[0].focus();
+    buttons[0].blur();
+
+    switch (event.key) {
```
eventuell klassen?

RebeccaRotti commented on game/index.html:
```diff
@@ -0,0 +1,52 @@
+    </header>
+    <main>
+        <div id="loose">You triggered the Alarm!</div>
+        <section class="game">
```
section ist immer in einem article
als hier statt section article verwenden

RebeccaRotti reviewed (commented):
sehr cool
würd dein projekt gern auf webmonster veröffentlichen mit deinem namen
ok oder lieber anonym oder gar nicht?
machst du noch was bezüglich mobil optimierung?

NoranWinterwolf committed:
game.html:
- change 'section' to 'article'

style.css:
- adding classes for javaScript

script.js:
- key listener now changes the button class instead of inline styling
- remove 'outputString' in checkCode()-function
- checkCode() now changes the element classes instead of inline styling

NoranWinterwolf committed:
- fixed #-button style after glowing
